src/main/groovy/com/github/livepvrdata/monitors/espn/json/MotorSportMonitor.groovy

```diff
@@ -51,12 +51,12 @@ abstract class MotorSportMonitor extends EventMonitor {
 		}
 	}
 
-	static final Event[] getFeed(String sport, String league) throws IOException {
+	static final Event[] getFeed(String sport, String league, String date) throws IOException {
 		String key = String.format("espnjson_%s_%s", sport, league)
 		def events = AppRuntime.instance.statusCache.get(key)
 		if(events == null) {
 			try {
-				def data = new URL(String.format(FEED_URL, sport, league)).text
+				def data = new URL(String.format(FEED_URL, sport, league, new Date(date).format(matchFmt, FEED_TZ)) + "&disable=links,competitors,broadcasts&dates=" + date)).text

```

Comment:
You're adding a new Date() string as an argument to String.format(), but since you didn't update the FEED_URL to have a third %s in it, that's not doing anything.  I suggest updating the FEED_URL to handle the disable-links and the date there.